app/src/main/res/layout/main_activity.xml

```diff
@@ -25,26 +25,14 @@
         android:layout_height="match_parent"
         tools:context=".MainActivity">
 
-        <com.google.android.material.navigationrail.NavigationRailView
-            android:id="@+id/navigation_rail"
-            android:layout_width="wrap_content"
-            android:layout_height="0dp"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            app:headerLayout="@layout/navigation_rail_header"
-            app:isGone="@{!@bool/show_navigation_rail}"
```

Comment:
Are we still using `@bool/show_navigation_rail` anywhere? If not, let's remove the resource.

Reply:
The boolean value is still used for task_detail.xml and the FAB in task_detail_fragment.xml. 

I'll update those two as well to be resource qualified.